src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/_params.py

```diff
@@ -0,0 +1,48 @@
+from azure.cli.command_modules.documentdb.sdk.models.location import Location
+
+
+def validate_failover_policies(ns):
+    ''' Extracts multiple space-separated failoverPolcies in region=failoverPriority format '''
+    fp_dict = []
+    for item in ns.failover_policies:
+        comps = item.split('=', 1)
+        fp_dict.append(FailoverPolicy(comps[0], int(comps[1])))
+    ns.failover_policies = fp_dict
+
+def validate_locations(ns):
+    ''' Extracts multiple space-separated locations in regionName=failoverPriority format '''
+    loc_dict = []
+    for item in ns.locations:
+        comps = item.split('=', 1)
+        loc_dict.append(Location(location_name=comps[0], failover_priority=int(comps[1])))
+    ns.locations = loc_dict
+
+register_cli_argument('documentdb', 'account_name', arg_type=name_type, help='Name of the DocumentDB Database Account', completer=get_resource_name_completion_list('Microsoft.DocumentDb/databaseAccounts'), id_part="name")
+
+register_cli_argument('documentdb regenerate-key', 'key_kind', **enum_choice_list(KeyKind))
+register_cli_argument('documentdb failover-priority-change', 'failover_policies', validator=validate_failover_policies, help="space separated failover policies in 'regionName=failoverPriority' format. E.g \"East US\"=0", nargs='+')
+
+register_cli_argument('documentdb create', 'account_name', completer=None)
+register_cli_argument('documentdb create', 'resource_group_name', help="name of the resource group")
+register_cli_argument('documentdb create', 'resource_group_location', ignore_type, help="location of the resource group")
+register_cli_argument('documentdb create', 'locations', validator=validate_locations, help="space separated locations in 'regionName=failoverPriority' format. E.g \"East US\"=0. Failover priority values are 0 for write regions and greater than 0 for read regions. A failover priority value must be unique and less than the total number of regions.", nargs='+')
+register_cli_argument('documentdb create', 'default_consistency_level', help="default consistency level of the DocumentDB Database account", **enum_choice_list(DefaultConsistencyLevel))
+register_cli_argument('documentdb create', 'max_staleness_prefix', help="when used with Bounded Staleness consistency, this value represents the number of stale requests tolerated. Accepted range for this value is 1 - 2,147,483,647", type=int)
+register_cli_argument('documentdb create', 'max_interval', help="when used with Bounded Staleness consistency, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100", type=int)
+register_cli_argument('documentdb create', 'ip_range_filter', help="firewall support. Specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs for a given database account. IP addresses/ranges must be comma separated and must not contain any spaces")
```

Comment:
We do not accept comma separated lists anywhere in the CLI. All of our lists are space-separated. If the SDK accepts a string with comma separated values, then there should be a validator on this parameter that takes the list of space-separated values and joins them with ','.